components/footers.js

```diff
@@ -87,6 +87,7 @@ export function Footer({ recentPost }) {
             </FooterLink>
             <FooterLink href="/pacer-facts/">PACER Fact Sheet</FooterLink>
             <FooterLink href="/idb-facts/">Integrated Database Fact Sheet</FooterLink>
+            <FooterLink href="/projects/filing-and-case-management/">Filing and Case Management System Research</FooterLink>
```

Comment:
Could we get away with this, if we remove the blog links:

```suggestion
            <FooterLink href="/projects/filing-and-case-management/">Court Contracts</FooterLink>
```